janusgraph-berkeleyje/src/main/java/org/janusgraph/diskstorage/berkeleyje/BerkeleyJEKeyValueStore.java

```diff
@@ -235,6 +250,13 @@ public void delete(StaticBuffer key, StoreTransaction txh) throws BackendExcepti
         }
     }
 
+    private static int convertSecondsToHoursTtl(final Integer ttl) {
+        if (hoursIsSeconds) {
```

Comment:
I don't understand why this special handling for tests is necessary. Could you please add a comment to explain why we need this (if we really do)?

Reply:
BerkeleyDB supports only hour-discrete ttl. It need for compatibility with testing framework. I deleted testing specific logic from class except annotation `@VisibleForTesting`